### ISSUE TITLE
When SSHing into the Windows VM I cannot choose a drive other than `C:`

### ISSUE THREAD

**Issue by Denis-Gavrielov** [microsoft/vscode-remote-release#5001]
- VSCode Version: Version: 1.55.1
- Local OS Version: MacOS 10.15.7
- Remote OS Version: Windows 10.0.18363
- Remote Extension/Connection Type: SSH
- Logs: N/A
Please note: I have downgraded my VS Code "Remote - SSH" extension to v0.62.0. Otherwise I would see similar bugs to what was reported [here](https://github.com/microsoft/vscode-remote-release/issues/4354).

Steps to Reproduce:

1. Launch a Windows VM with Parallels
2. Mount your Host directories in the Windows VM "Z" drive
3. Try to open that drive when connection to the Windows VM via the VS Code SSH extension. It seems like choosing any other path than something that starts with `/` or `c:` is invalid. 

Does this issue occur when you try this locally?: N/A -> when opening VS Code in my Windows VM I can open a folder in my "Z" drive.
Does this issue occur when you try this locally and all extensions are disabled?: N/A

Screenshot of me attempting to open the "Z" drive:
![image](https://user-images.githubusercontent.com/31205684/117028357-8dbef880-acf5-11eb-8f50-491a438af11a.png)
![image](https://user-images.githubusercontent.com/31205684/117028405-99aaba80-acf5-11eb-9ecf-dcd9f8886784.png)

**Comment by Denis-Gavrielov** [comment]
Hi @alexr00, I have now had the same issue with VS Code version 1.55 and 1.56. 

Please note that my Host OS is MacOS. Not sure if that matters.

How are you opening other drives? Can you maybe share a screenshot of how you do it in the ssh'd windows VM?

**Comment by Denis-Gavrielov** [comment]
Hi @alexr00, any updates on this?

**Comment by Denis-Gavrielov** [comment]
Hi @alexr00, can you please remove the "needs-more-info" label and get back to this? I am still unsure how you have accessed a different drive when sshing into the windows VM.

**Comment by alexr00** [comment]
@Denis-Gavrielov I was using a Windows -> Windows connection, so that might have something to do with it. I don't have macOS to test with, but I still plan on testing with Ubuntu -> Windows.

**Comment by alexr00** [comment]
I tested Ubuntu -> Windows, and I was able to open a folder from a different drive, but it was pretty buggy. Path separators moved around, quick pick items didn't select. It definitely needs work. 

![recording (58)](https://user-images.githubusercontent.com/38270282/121189641-92ce1500-c86a-11eb-8f9f-196ee0d8ecb8.gif)

I'll try to fix the issues I encountered, and we'll see that also fixes the drive issue that you're seeing.
